app/lib/discourse_automation/scripts/close_topic.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+DiscourseAutomation::Scriptable::CLOSE_TOPIC = 'close_topic'
+
+DiscourseAutomation::Scriptable.add(DiscourseAutomation::Scriptable::CLOSE_TOPIC) do
+  field :topic, component: :text, required: true
+  field :message, component: :text
+  field :user, component: :user
+
+  version 1
+
+  # FIXME: copied over from pin_topic; needs some thought
```

Comment:
Maybe stalled_wiki could be added here ?

Reply:
It was more like a PONDERME than a FIXME; I'm not that familliar with the code to know from which other triggerables it would make sense to allow this new script. We can consider this pondered now :)